src/documentation/pages/components/avatar/examples/_icon-avatar.vue

```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Icon Avatars" id="icon-avatars" :tabs="exampleTabs">
+    p.-text(slot="example-description")
+      | Icon avatars can be used when an image is not available.
+      | To use, wrap any <a href="../icon/">Chi icon</a> with <code>chi-avatar</code>.
```

Comment:
Be careful with redirection
```suggestion
      | To use, wrap any <a href="/components/icon/">Chi icon</a> with <code>chi-avatar</code>.
```